src/components/PluginsPane.js

```diff
@@ -4,14 +4,17 @@ import PropTypes from 'prop-types';
 import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
 
 export class PluginsPane extends React.Component {
-  static propTypes = [{
-    plugins: PropTypes.arrayOf(
-      PropTypes.shape({
-        title: PropTypes.string,
-        content: PropTypes.any
-      })
-    )
-  }];
+  static propTypes = [
+    {
+      plugins: PropTypes.arrayOf(
+        PropTypes.shape({
+          title: PropTypes.string,
+          plugin: PropTypes.func.isRequired,
+        }),
+      ),
+      value: PropTypes.string,
```

Comment:
What is this?

Reply:
This is a prop that accepts the results value as a string. It's was sent to the current `ResultsViewer` component that renders the results json obj. 
We're passing it into the PluginsPane to share it with all the plugins. It's used here: 
https://github.com/GraphQL-Query-Planner/graphiql/blob/extract-plugin/src/components/GraphiQL.js#L389
A few lines above you can see the same pattern is used to share the results with the `ResultViewer`